src/onegov/form/parser/snippets.py

```diff
@@ -39,6 +39,13 @@ class Snippets:
             '    [ ] B\n'
             '    [ ] C'
         )),
+        (_("Subfields depending on selected option"), (
```

Comment:
```suggestion
        (_("Subfields depending on choice"), (
```

This wording is a little bit more consistent

Reply:
I like it, will adapt